features/firewall/README.md

```diff
@@ -1,19 +1,13 @@
 # Feature: cis_firewall
```

Comment:
Since the firewall feature isn't directly related to CIS anymore, we should adjust the header here and change it to `firewall` then.

Reply:
Thanks for pointing this out, I updated the header.